### PR TITLE
HPC: Move single-machine HPC tests to yaml scheduler

### PR DESCRIPTION
**Pull request by schlad**
- Related ticket: https://progress.opensuse.org/issues/65753
- Verification run:
**x86_64:**
rasdaemon: https://openqa.suse.de/tests/4172091
powerman: https://openqa.suse.de/tests/4172093
genders: https://openqa.suse.de/tests/4172228
conman: https://openqa.suse.de/tests/4172230
**aarch64:**
rasdaemon: https://openqa.suse.de/tests/4172092
powerman: https://openqa.suse.de/tests/4172094
genders: https://openqa.suse.de/tests/4172229
conman: https://openqa.suse.de/tests/4172231

### CODE REVIEW

File: schedule/hpc/single_machine_test.yaml [hunk 1]
```diff
@@ -0,0 +1,31 @@
+---
+name: single machine HPC tests
+description:    >
+     Maintainer: schlad
+     all singe machine HPC tests based loaded based on hpctest
+vars:
+  DESKTOP: textmode
+  INSTALLONLY: 1
```
Comment (czerw) [inline schedule/hpc/single_machine_test.yaml:8]: INSTALLONLY=1 is needed for installation jobs only

Comment (schlad) [reply schedule/hpc/single_machine_test.yaml]: ah sure. re-sent.

**Comment by czerw** [comment]
Wouldn't be better to schedule all tests one by one in one test suite/job? And avoid current situation of one test  = one job.

**Comment by schlad** [comment]
> Wouldn't be better to schedule all tests one by one in one test suite/job? And avoid current situation of one test = one job.

very good point. This of course crossed my mind too.
for now not all those tests run using virtio.

Anyway after thinking about this today morning I assumed it is better to keep it as it is for SP2 and do this in the next step:
https://progress.opensuse.org/issues/65022

This will completely change the look of the "job group' as I see it;
we would have:
-alpha_cluster
- beta_cluster
...

and single test of say rasdaemon would be added to any/all/some of those.

So I don't mind not migrating to yaml for hpc, work instead on the poo#65022 from the start using yaml there.

**Comment by czerw** [comment]
> Anyway after thinking about this today morning I assumed it is better to keep it as it is for SP2 and do this in the next step:
> https://progress.opensuse.org/issues/65022

Ok, let's keep it like it is and we can have major change for sp3.

**Pull request merged**